### ISSUE TITLE
snap: arm64: cannot build qemu

### ISSUE THREAD

**Issue by devimc** [kata-containers/kata-containers#2194]
```
INFO: Apply /build/kata-containers/stage/gopath/src/github.com/kata-containers/kata-containers/tools/packaging/qemu/patches/5.2.x/0010-Merge-remote-tracking-branch-remotes-ehabkost-gl-tag.patch
+ /build/kata-containers/stage/gopath/src/github.com/kata-containers/kata-containers/tools/packaging/scripts/apply_patches.sh /build/kata-containers/stage/gopath/src/github.com/kata-containers/kata-containers/tools/packaging/qemu/patches/tag_patches/v5.2.0
INFO: Apply patches from /build/kata-containers/stage/gopath/src/github.com/kata-containers/kata-containers/tools/packaging/qemu/patches/tag_patches/v5.2.0
INFO: Found 0 patches
++ uname -m
+ '[' ppc64le = x86_64 ']'
+ configure_hypervisor=/build/kata-containers/stage/gopath/src/github.com/kata-containers/kata-containers/tools/packaging/scripts/configure-hypervisor.sh
+ chmod +x /build/kata-containers/stage/gopath/src/github.com/kata-containers/kata-containers/tools/packaging/scripts/configure-hypervisor.sh
+ xargs ./configure
++ PREFIX=/snap/kata-containers/current/usr
++ /build/kata-containers/stage/gopath/src/github.com/kata-containers/kata-containers/tools/packaging/scripts/configure-hypervisor.sh -s kata-qemu
+ echo '--disable-sheepdog --disable-live-block-migration --disable-brlapi --disable-docs --disable-curses --disable-gtk --disable-opengl --disable-sdl --disable-spice --disable-vte --disable-vnc --disable-vnc-jpeg --disable-vnc-png --disable-vnc-sasl --disable-auth-pam --disable-glusterfs --disable-libiscsi --disable-libnfs --disable-libssh --disable-bzip2 --disable-lzo --disable-snappy --disable-tpm --disable-slirp --disable-libusb --disable-usb-redir --static --disable-debug-tcg --disable-tcg-interpreter --disable-qom-cast-debug --disable-tcmalloc --disable-curl --disable-rdma --disable-tools --enable-virtiofsd --enable-virtfs --disable-bsd-user --disable-linux-user --disable-sparse --disable-vde --disable-xfsctl --disable-libxml2 --disable-nettle --disable-xen --disable-linux-aio --disable-capstone --disable-virglrenderer --disable-replication --disable-smartcard --disable-guest-agent --disable-guest-agent-msi --disable-vvfat --disable-vdi --disable-qed --disable-qcow1 --disable-bochs --disable-cloop --disable-dmg --disable-parallels --enable-kvm --enable-vhost-net --enable-virtfs --enable-attr --enable-cap-ng --enable-seccomp --disable-avx2 --disable-libpmem --enable-malloc-trim --target-list=ppc64-softmmu --extra-cflags=" -O2 -fno-semantic-interposition -falign-functions=32 -D_FORTIFY_SOURCE=2 -fPIE" --extra-ldflags=" -z noexecstack -z relro -z now" --prefix=/snap/kata-containers/current/usr --libdir=/snap/kata-containers/current/usr/lib/kata-qemu --libexecdir=/snap/kata-containers/current/usr/libexec/kata-qemu --datadir=/snap/kata-containers/current/usr/share/kata-qemu    --disable-rbd '
Using './build' as the directory for build output

ERROR: User requested feature libseccomp
       configure was not able to find it.
       Install libseccomp devel >= 2.3.0

Failed to run 'override-build': Exit code was 123.
Build failed
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/lpbuildd/target/build_snap.py", line 230, in run
    self.pull()
  File "/usr/lib/python3/dist-packages/lpbuildd/target/build_snap.py", line 196, in pull
    env=env)
  File "/usr/lib/python3/dist-packages/lpbuildd/target/build_snap.py", line 100, in run_build_command
    return self.backend.run(args, env=full_env, **kwargs)
  File "/usr/lib/python3/dist-packages/lpbuildd/target/lxd.py", line 537, in run
    subprocess.check_call(cmd, **kwargs)
  File "/usr/lib/python3.6/subprocess.py", line 311, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['lxc', 'exec', 'lp-focal-ppc64el', '--env', 'LANG=C.UTF-8', '--env', 'SHELL=/bin/sh', '--env', 'SNAPCRAFT_LOCAL_SOURCES=1', '--env', 'SNAPCRAFT_SETUP_CORE=1', '--env', 'SNAPCRAFT_BUILD_INFO=1', '--env', 'SNAPCRAFT_IMAGE_INFO={"build-request-id": "lp-64607377", "build-request-timestamp": "2021-07-07T16:06:18Z", "build_url": "https://launchpad.net/~build.snapcraft.io/+snap/029f6c9452eb983c086114e2929f7413/+build/1458686"}', '--env', 'SNAPCRAFT_BUILD_ENVIRONMENT=host', '--env', 'http_proxy=http://10.10.10.1:8222/', '--env', 'https_proxy=http://10.10.10.1:8222/', '--env', 'GIT_PROXY_COMMAND=/usr/local/bin/snap-git-proxy', '--', '/bin/sh', '-c', 'cd /build/kata-containers && linux64 snapcraft pull']' returned non-zero exit status 2.
Revoking proxy token...
RUN: /usr/share/launchpad-buildd/bin/in-target scan-for-processes --backend=lxd --series=focal --arch=ppc64el SNAPBUILD-1458686
Scanning for processes to kill in build SNAPBUILD-1458686

```

cc @jongwu  @fidencio

**Comment by jongwu** [comment]
hello @devimc -, what's the problem with arm from the log? can you offer the link here?

**Comment by devimc** [comment]
@jongwu we are trying to re-enable the snap for all architectures, we are using the snapcraft.io build system (https://snapcraft.io/blog/we-are-changing-the-way-you-build-snaps-from-github-repos) for that,  @Jakob-Naucke  is fixing s390x (and maybe ppc64) issues and I thought that you can help us with arm64, you can easily reproduce this issue by running `snapcraft build` in the root of this repo

**Comment by Jakob-Naucke** [comment]
So the error in the OP is probably fixed by #2193, but there might be more which I cannot test right now.

**Comment by jongwu** [comment]
Thanks @devimc @Jakob-Naucke -， I will enable snap build on arm.
